@vates/fuse-vhd/.USAGE.md

```diff
@@ -0,0 +1,28 @@
+Mount a vhd generated by xen-orchestra to filesystem
```

Comment:
FYI (I'm not requesting any change here :slightly_smiling_face:), if you put a `description` in the `package.json`, it will be automatically added in the README: https://github.com/vatesfr/xen-orchestra/tree/master/%40vates/async-each#vatesasync-each